packages/models-library/src/models_library/projects_nodes.py

```diff
@@ -107,8 +26,7 @@ class Config:
     int, bool, str, float, SimCoreFileLink, DatCoreFileLink, DownloadLink
 ]
 
-InputID = constr(regex=PROPERTY_KEY_RE)
-OutputID = InputID
+InputID = OutputID = constr(regex=PROPERTY_KEY_RE)
 Inputs = Dict[InputID, InputTypes]
 Outputs = Dict[OutputID, OutputTypes]
```

Comment:
no ```__root__``` here?

Reply:
nope... i had some bad experiences with that in the workbench (see comments in projects.py) . I first need to understand it better before using it